workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/CamundaExecutor.java

```diff
@@ -147,6 +147,11 @@ public void setOutputVariable(String name, Object value) {
       execution.setVariable(String.format("%s.%s.%s", activityId, ActivityExecutorContext.OUTPUTS, name), valueVar);
     }
 
+    @Override
+    public Map<String, Object> getVariables() {
+      return (Map<String, Object>) execution.getVariable(VARIABLES);
```

Comment:
let's maybe return a copy of that to avoid using it as a way to change variables (not sure if it would be possible though)

Reply:
Assuming we use copies, what if the variable changes during workflow execution (with a script), do we want to keep the first value (that is in the copy) or the up-to-date one?

Reply:
What about prefixing template related variables or putting them in a dedicated object so we don't mix them with regular variables. 
Like this a regular variable can be used with:  **_variables.myVariable_**, while template's ones can be used with: **_variables.templates.myVariable_**  or even **_variables.templates.SOME_ID_RELATED_TO_FILE.myVariable_**. 

Does it worth doing this or not?